cogs/search.py

```diff
@@ -0,0 +1,297 @@
+            """Look up manga information."""
+
+            base = "https://kitsu.io/api/edge/"
+
+            # Handling
+            async with ctx.typing():
+                async with aiohttp.ClientSession() as session:
+                    async with session.get(base + "manga", params={"filter[text]": query}) as resp:
+
+                        resp = await resp.json()
+                        resp = resp["data"]
+
+                        query = utils.escape_mentions(query)
+                        query = utils.escape_markdown(query)
+
+                        if not resp:
+                            await ctx.send(f"No results for `{query}`.")
+                            return
+
+                        manga = resp[0]
+                        title = f'{manga["attributes"]["canonicalTitle"]}'
+                        manga_id = manga["id"]
+                        url = f"https://kitsu.io/manga/{manga_id}"
+
+                        embed = Embed(title=f"{title}", color=ctx.author.color, url=url)
+                        embed.description = manga["attributes"]["synopsis"][0:425] + "..."
+                        if manga["attributes"]["averageRating"]:
+                            embed.add_field(name="Average Rating", value=manga["attributes"]["averageRating"])
+                        embed.add_field(name="Popularity Rank", value=manga["attributes"]["popularityRank"])
+                        if manga["attributes"]["ageRating"]:
+                            embed.add_field(name="Age Rating", value=manga["attributes"]["ageRating"])
+                        embed.add_field(name="Status", value=manga["attributes"]["status"])
+                        thing = "" if not manga["attributes"]["endDate"] else f' to {manga["attributes"]["endDate"]}'
+                        embed.add_field(name="Published", value=f"{manga['attributes']['startDate']}{thing}")
+                        if manga["attributes"]["chapterCount"]:
+                            embed.add_field(name="Chapters", value=manga["attributes"]["chapterCount"])
+                        embed.add_field(name="Type", value=manga["attributes"]["mangaType"])
+                        embed.set_thumbnail(url=manga["attributes"]["posterImage"]["original"])
+
+                        try:
+
+                            await ctx.send(f"**{title}** - <{url}>", embed=embed)
+
+                        except Exception:
+
+                            aired = f"{manga['attributes']['startDate']}{thing}"
+                            template = f"""```
+            url: {url}
+            Title: {title}
+            Average Rating: {manga["attributes"]["averageRating"]}
+            Popularity Rank: {manga["attributes"]["popularityRank"]}
+            Age Rating: {manga["attributes"]["ageRating"]}
+            Status: {manga["attributes"]["status"]}
+            Aired: {aired}
+            Type: {manga['attributes']["showType"]}
+            Powered by kitsu.io```"""
```

Comment:
```suggestion
                            template = textwrap.dedent(
                                f"""
                                ```
                                url: {url}
                                Title: {title}
                                Average Rating: {manga["attributes"]["averageRating"]}
                                Popularity Rank: {manga["attributes"]["popularityRank"]}
                                Age Rating: {manga["attributes"]["ageRating"]}
                                Status: {manga["attributes"]["status"]}
                                Aired: {aired}
                                Type: {manga['attributes']["showType"]}
                                Powered by kitsu.io
                                ```
                                """
                            )
```